src/App.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <router-view />
+  <PlayerBrowserMediaControls />
```

Comment:
What would be the downside of having this component always loaded ? Does it block playback of other browser tabs ? I think not right ? So it should be safe to just enable this by default for everyone and if somebody starts complaining, we can add a toggle to disable ?

Reply:
on phones it could be that it blocks the sound from other sources. on PCs it shouldnt. i would just go with it and fix it if someone is complaining